example/bundle.js

```diff
@@ -0,0 +1,11 @@
+const bundle = require('@asyncapi/bundler');
+const fs = require('fs');
+
+async function main(){
```

Comment:
I would suggest that you incorporate this example into a test that can be run on each PR as a form of system test 🙂

Cause just having an example, does not mean the example continuous to work.

Reply:
I did not keep the old logic 😅 , let me put it in the option